### PR TITLE
Bump version to 0.5.0

### PR DESCRIPTION
**Pull request by DominikWolek**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -40,11 +40,12 @@ defmodule Membrane.FLV.Mixfile do
...
       {:membrane_mp4_format, "~> 0.7.0"},
-      {:membrane_h264_format, "~> 0.4.0"},
+      {:membrane_h264_format, "~> 0.5.0"},
       {:membrane_file_plugin, "~> 0.13.2", only: :test},
-      {:membrane_h264_ffmpeg_plugin, "~> 0.25.1", only: :test},
+      {:membrane_h264_ffmpeg_plugin, "~> 0.25.3", only: :test},
       {:membrane_aac_plugin, "~> 0.13.0", only: :test},
-      {:membrane_mp4_plugin, "~> 0.18.0", only: :test},
+      {:membrane_mp4_plugin,
+       github: "membraneframework/membrane_mp4_plugin", branch: "bump-dependencies", only: :test},
```
Comment (daniel-jodlos) [inline mix.exs:48]: It's a test dependency, so we technically could merge it like that, but I'd rather not. Please change it to hex dependency before you merge

**Review by daniel-jodlos — APPROVED** [review]

File: mix.exs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.FLV.Mixfile do
   use Mix.Project
 
-  @version "0.4.0"
+  @version "0.4.1"
```
Comment (varsill) [inline mix.exs:4]: Hmm I think we are breaking the API since now we will be using different h264 format - shouldn't we bump the minor?

Comment (daniel-jodlos) [reply mix.exs:4]: The change in H264 format, according to the diff, is only making one of the fields optional. Plugins upgrading from the previous version should be setting said field, so unless it changes in the update, this is not a breaking change of their API.

Comment (varsill) [reply mix.exs:4]: Well, then probably we shouldn't have bumped the minor in h264_format then :D

**Review by varsill — APPROVED** [review]

**Pull request merged**